generator/template/process.go

```diff
@@ -169,9 +172,38 @@ func processTableSQLBuilder(fileTypes, dirPath string,
 
 		err = utils.SaveGoFile(tableSQLBuilderPath, tableSQLBuilder.FileName, text)
 		throw.OnError(err)
+
+		generatedBuilders = append(generatedBuilders, tableSQLBuilder)
+	}
+
+	if len(generatedBuilders) > 0 {
+		generateUseSchemaFunc(dirPath, fileTypes, schemaMetaData, generatedBuilders)
 	}
 }
 
+func generateUseSchemaFunc(dirPath, fileTypes string, schemaMetadata metadata.Schema, builders []TableSQLBuilder) {
+
+	basePath := path.Join(dirPath, builders[0].Path)
```

Comment:
tableSQLBuilderPath can be passed from the caller.

Reply:
Ignore this. I just noticed tableSQLBuilderPath is inside a loop.

Reply:
@go-jet good point actually. `generateUseSchemaFunc` can simply accept a single `TableSQLBuilder`, passed down by `processTableSQLBuilder`. The invocation could look like `generateUseSchemaFunc(dirPath, fileTypes, generatedBuilders[0])`.